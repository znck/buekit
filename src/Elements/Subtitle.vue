<script>
import createMixin from './title/mixin'

export default {
  name: 'Subtitle',
  ...createMixin('subtitle')
}
</script>

<style lang="scss" module>
@import '~bulma/sass/utilities/_all.sass';
@import '../_base.scss';
@import '~bulma/sass/elements/title.sass';
</style>
