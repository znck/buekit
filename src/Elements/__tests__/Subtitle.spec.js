import {shallow} from 'vue-test-utils'
import {createRenderer} from 'vue-server-renderer'

import Subtitle from '../Subtitle.vue'

describe('Subtitle.vue', () => {
	it('matches snapshot', () => {
		const wrapper = shallow(Subtitle, {})
		const renderer = createRenderer()

		renderer.renderToString(wrapper.vm, (err, str) => {
			if (err) {
				throw new Error(err)
			}
			expect(str).toMatchSnapshot()
		})
	})
})
