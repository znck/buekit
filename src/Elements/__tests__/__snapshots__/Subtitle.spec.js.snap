// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle.vue matches snapshot 1`] = `<h1 class="subtitle"></h1>`;
